.github/scripts/link-check.sh

```diff
@@ -0,0 +1,9 @@
+# Actions invokes script with: bash -e <script>
+
+set -a
+
+source $(dirname ${BASH_SOURCE[0]})/common.sh
+ci_conda_activate
+cd docs
+source install-deps
+make linkcheck
```

Comment:
This script runs the Sphinx link checker in CI to verify correctness of URLs in the documentation.